### PR TITLE
fix: Table that expresses all HTTP responses returned by the admin API

### PR DESCRIPTION
**Pull request by cloudjumpercat**
### Description

Customers were requesting a table that explains which status codes the admin API returns and what those codes mean.
 

DOCU-2952
### Testing instructions

Preview link: 

### Checklist 

- [x] Review label added 
- [x] [Conditional version tags](https://docs.konghq.com/contributing/conditional-rendering/#conditionally-render-content-by-version) added, if applicable.

### CODE REVIEW

File: app/_src/gateway/admin-api/index_latest.md [hunk 1]
```diff
@@ -146,6 +146,21 @@ curl -i -X POST http://localhost:8001/services/test-service/routes \
      -F "paths[2]=/path/two"
 ```
 
+## HTTP status response codes
+
+The following status codes are returned in HTTP responses:
+
+| HTTP Code | HTTP Description | Notes | Request method |
+| --------- | ---------------- | ----- | ------------- |
+| 200 | OK | The request succeeded. The result of a `200` request depends on the request type: `GET`: The resource was fetched and sent in the message body. `PUT` or `POST`:  The resource that describes the result of the action is sent in the message body. `PATCH`: ? | `GET`, `POST`, `PATCH`, `PUT` |
```
Comment (lena-larionova) [inline app/_src/gateway/admin-api/index_latest.md:155]:
```suggestion
| 200 | OK | The request succeeded. The result of a `200` request depends on the request type: <br>- `GET`: The resource was fetched and sent in the message body. <br>- `PUT` or `POST`:  The resource that describes the result of the action is sent in the message body. <br>- `PATCH`: ? | `GET`, `POST`, `PATCH`, `PUT` |
```

Comment (cloudjumpercat) [reply app/_src/gateway/admin-api/index_latest.md:155]: @lena-larionova Thanks! I couldn't remember how we formatted those

File: app/_src/gateway/admin-api/index_latest.md [hunk 2]
```diff
@@ -146,6 +146,21 @@ curl -i -X POST http://localhost:8001/services/test-service/routes \
      -F "paths[2]=/path/two"
 ```
 
+## HTTP status response codes
+
+The following status codes are returned in HTTP responses:
+
+| HTTP Code | HTTP Description | Notes | Request method |
+| --------- | ---------------- | ----- | ------------- |
+| 200 | OK | The request succeeded. The result of a `200` request depends on the request type: <br>- `GET`: The resource was fetched and sent in the message body. <br>- `PUT` or `POST`:  The resource that describes the result of the action is sent in the message body. <br>- `PATCH`: ? | `GET`, `POST`, `PATCH`, `PUT` |
```
[the remark this answers is not in the document]
Comment (lena-larionova) [reply app/_src/gateway/admin-api/index_latest.md:155]: That sounds right, but pinging @Guaris for the real answer.

**Review by lena-larionova — APPROVED** [review]
LGTM, if the PATCH description is correct.

File: app/_src/gateway/admin-api/index_latest.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (Guaris) [reply app/_src/gateway/admin-api/index_latest.md:155]: thats right

**Pull request merged**